darwin/future/data_objects/workflow.py

```diff
@@ -173,7 +179,7 @@ class Workflow(DefaultDarwin):
     inserted_at: datetime
     updated_at: datetime
 
-    dataset: WFDataset
+    dataset: Optional[WFDataset]
```

Comment:
Some workflows on platform don't have a dataset attached, prevents collecting these items if it's required field. Although intuitively you would think it should be

Reply:
Yep, intuitively I _did_ think so!  Good catch